tests/modules/test_measureimageareaoccupied.py

```diff
@@ -1,6 +1,7 @@
 import unittest
 
 import numpy
+import StringIO
```

Comment:
You should consider using the StringIO package available from `six` to ease the Python 3 transition.